Gemfile

```diff
@@ -7,6 +7,10 @@ end
 
 gemspec
 
+group :dtls do
```

Comment:
The file `lib/david.rb` contains the following lines:

```ruby
require 'bundler/setup'
Bundler.require(:default, :cbor)
```

Should I add `Bundler.require(:default, :dtls)` to that file as well?